grails-app/i18n/messages.properties

```diff
@@ -4801,9 +4801,14 @@ react.cycleCount.indicators.cardTitle.totalCount.label=Total count physical inve
 react.cycleCount.indicators.cardTitle.itemsCounted.label=Items counted
 react.cycleCount.indicators.cardTitle.targetProgress.label=Target progress
 react.cycleCount.indicators.cardTitle.notFinishedItems.label=Not finished items
+react.cycleCount.indicators.cardTitle.inventoryLoss.label=Inventory Loss
 react.cycleCount.indicators.cardSubtitle.inMyInventory.label=in my inventory
 react.cycleCount.indicators.cardSubtitle.itemsCounted.label=items counted
+react.cycleCount.indicators.cardSubtitle.countedLess.label=products counted less than in OB
+react.cycleCount.indicators.cardSubtitle.overallLost.label=overall financial value lost
 react.cycleCount.indicators.cardInfo.totalCount.label=Total physical inventory count
 react.cycleCount.indicators.cardInfo.itemsCounted.label=Total items counted
 react.cycleCount.indicators.cardInfo.targetProgress.label=Shows how far along the count progress is
 react.cycleCount.indicators.cardInfo.notFinishedItems.label=Items remaining to be counted or processed
+react.cycleCount.indicators.cardInfo.inventoryLoss.label=Inventory loss count is the number of products for which the overall inventory change over the time range selected is a negative adjustment (qty at the beginning of the time range is more than qty at the end). The financial inventory loss is the equivalent of the inventory decrease for these products in USD.
+react.cycleCount.indicators.million.label=million
```

Comment:
It looks like it's not a cycle count-related label; it can be put under `default`